tests/auth_tests/test_signals.py

```diff
@@ -97,10 +97,15 @@ def test_login_with_custom_user_without_last_login_field(self):
             signals.user_logged_in.receivers = []
             with self.assertRaises(FieldDoesNotExist):
                 MinimalUser._meta.get_field('last_login')
+
             with self.settings(AUTH_USER_MODEL='auth_tests.MinimalUser'):
                 apps.get_app_config('auth').ready()
             self.assertEqual(signals.user_logged_in.receivers, [])
 
+            with self.settings(AUTH_USER_MODEL='auth_tests.UserWithDisabledLastLoginField'):
+                apps.get_app_config('auth').ready()
+            self.assertEqual(signals.user_logged_in.receivers, [])
+
             with self.settings(AUTH_USER_MODEL='auth.User'):
                 apps.get_app_config('auth').ready()
             self.assertEqual(len(signals.user_logged_in.receivers), 1)
```

Comment:
These checks aren't quite independent. When this one is first it fails. But I guess it's OK.

Reply:
Personally, I would not like to change an existing test, as it checking some fixed behavior. In my mind, it's different cases. If I've screwed up with a new condition, the first test will tell me about it.

Reply:
No problem. I think it’s OK as is. (Adjusting it would be done separately anyway)

Reply:
@carltongibson can you please check it?

Reply:
Yep. Looks fine. I've marked it Ready for Checkin. Most likely Tim will have a look and merge it if he's happy.